### DISCUSSION TITLE
Admin UI Dashboard Enhancements

### DISCUSSION THREAD

**Discussion by Layoric** [ServiceStack/Discuss#2] [Ideas]
A new feature in ServiceStack v6 was the `Admin UI` which currently manages users and has a simple dashboard. The UI was developed with the ability to incorporate new dashboards/screens that users would find most useful.

![image](https://user-images.githubusercontent.com/234642/151296405-ad97346e-f1b7-4f50-9c15-28a6476e21bb.png)

Some ideas we've had revolve around monitoring features for databases or API usage, but we'd be interested in what kind of features would make the biggest difference to your daily workflow and why? Is there something missing you would like to see in user management? Or can be something completely different related to common to administering your ServiceStack applications.

Interested to hear from community!

**Comment by LEIRONGHUA** [comment]
I hope to increase the display of API usage to facilitate interface current limiting and response time optimization.

**Reply by mythz** [reply]
Check out the new API Analytics in ServiceStack v5.7 https://docs.servicestack.net/admin-ui-analytics

**Comment by Drammy** [comment]
Is it possible to add our own content into the Dashboard?  I've some stats I'd like to add..

**Reply by mythz** [reply]
Have a look at [Custom UI docs](https://docs.servicestack.net/locode/custom-overview#admin-ui) for **/admin-ui**, you'll need to make a local modified copy of [Dashboard.mjs](https://github.com/ServiceStack/ServiceStack/blob/main/ServiceStack/src/ServiceStack/modules/admin-ui/components/Dashboard.mjs) and host it locally.
